docs/ldml/tr35-keyboards.md

```diff
@@ -2469,7 +2494,7 @@ TODO: Rewrite this? Probably push out to each element's section?
 3.  No `keyMap[@modifiers]` value can overlap with another `keyMap[@modifiers]` value.
     * eg you can't have `"RAlt Ctrl"` in one `keyMap`, and `"Alt Shift"` in another (because Alt = RAltLAlt).
 4.  Every sequence of characters in a `transform[@from]` value must be a concatenation of two or more `map[@to]` values.
-    * eg with `<transform from="xyz" to="q">` there must be some map values to get there, such as `<map... to="xy">` & `<map... to="z">`
+    * eg with `<transform from="xyz" to="q">` there must be some map values to get there, such as `<map… to="xy">` & `<map… to="z">`
```

Comment:
This looks like the element has a prefix "map", like "mapStuff". If you really mean intervening attributes, put a space between map and …

Reply:
Also in commented out section that needs to be fixed

Reply:
Why is this resolved?

Reply:
The entire commented out section was removed.